test/UnitTests/Microsoft.Testing.Platform.UnitTests/Logging/FileLoggerTests.cs

```diff
@@ -214,13 +214,11 @@ public async Task Log_WhenAsyncFlush_StreamWriterIsCalledOnlyWhenLogLevelAllowsI
             _mockFileStreamFactory.Object);
         fileLogger.Log(currentLogLevel, Message, null, Formatter, Category);
 
+        // Ensures that the async flush is completed before the file is read
+        fileLogger.Dispose();
```

Comment:
The variable is declared with `using`, so Dispose will be called twice. It's usually not a problem as the second dispose will simple `return;`, but it's redundant.

I would remove `using` from line 207, and wrap line 215 with `try/finally` and have the dispose call in finally.

The current code is still okay though and will not cause issues.